### ISSUE TITLE
maxlength of input with length of pattern

### ISSUE THREAD

**Issue by maraism** [BankFacil/vanilla-masker#62]
Hi,

I think when you apply pattern in input, you'll be set maxlength attribute on input with the length of pattern. 
Line 84 after 
this.elements[i].onkeyup = onType; 
Add 
this.elements[i].setAttribute('maxlength', this.opts.pattern.length); 

Thanks,
Maxime

**Comment by leandroadacosta** [comment]
@maraism yes. Cool, can be. We should do this for VM next version, thanks.
